types/lodash/index.d.ts

```diff
@@ -6366,98 +6366,34 @@ declare namespace _ {
         /**
          * @see _.forEach
          */
-        each<T>(
-            collection: T[],
-            iteratee?: ListIterator<T, any>
-        ): T[];
-
-        /**
-         * @see _.forEach
-         */
-        each(
-            collection: string,
+        each<TString extends string | null | undefined>(
+            collection: TString,
             iteratee?: ListIterator<string, any>
-        ): string;
+        ): TString;
 
         /**
          * @see _.forEach
          */
-        each<T>(
-            collection: List<T>,
+        each<T, TList extends List<T> | null | undefined>(
```

Comment:
You use not just `List<T> | null | undefined`, but `TList extends` that type, doesn't it? Of course if `TList extends List<T>` can be substituted, but using TList itself is more accurate, isn't it?